### PR TITLE
few tweaks / suggestions for multi-objective post

### PR DESCRIPTION
**Pull request by whoburg**
Was reading through the "Design is Multi-Objective" post and noticed a few places where the wording could be cleaned up. Feel free to accept or reject any of the suggestions.

Will also make a few inline comments about paragraphs that could still use some work.

### CODE REVIEW

File: _posts/2015-10-08-design-is-multi-objective.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ title: Design is Multi-Objective
...
 We need to explore a design space of competing alternatives, many of which have irreducable tradeoffs
 (e.g., between a fast short-hop plane and a slow long-distance one).
 The good news is that numerical optimization can take our complicated model straight to these tradeoffs;
 the bad is that even after that you still have to figure out which you want.
 
-Numerical optimization is often thought of as a way to “get an answer”.
+Numerical optimization is often thought of as a way to "get an answer".
 From a mathematical perspective, this is true;
 it’s like going to the Oracle with your linear program and asking the priests for the minimum possible value of $x$;
 after some deliberation, you get an answer you believe to be true: $3$.
```
Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:23]: The Oracle analogy is a little confusing / hard to follow. Try to clean up for readers that might not be familiar with the references you are making.

File: _posts/2015-10-08-design-is-multi-objective.md [hunk 2]
```diff
@@ -29,27 +29,27 @@ but inaccuracies in the model can result in absurdly unrealistic optima that are
 
 This might seem like a disadvantage:
 after all, you give the computer a model you trusted, and it spat out something ridiculous.
-But this is in fact a first  and vital step:
+But this is in fact a first and vital step:
 where you saw a series of trusted equations, the computer saw a hairball of emergent interconnections and told you so.
```
Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:33]: really like the ending of the previous sentence.

File: _posts/2015-10-08-design-is-multi-objective.md [hunk 3]
```diff
@@ -29,27 +29,27 @@ but inaccuracies in the model can result in absurdly unrealistic optima that are
...
 
 Optimization gives a way for a model to earn your trust;
-as it produces good and results with the cases you throw at it,
+as it produces good end results with the cases you throw at it,
 you understand where to trust your models and where they need improvement.
 Once you trust your model not just on paper, but in practice, you can use it to generate designs for comparison.
 
 So, with your tested model and multiple objectives (e.g., airplane range and speed), you want to start examining tradeoffs, and choosing between good designs.
-The variety of design possibilities is sometimes called the “design space”; but since you’ve decided to only care about certain variables (your objectives), what you want to see is each design plotted according to its merits, what I’ll call the “objective space”. [after Mueller]
+The variety of design possibilities is sometimes called the "design space"; but since you’ve decided to only care about certain variables (your objectives), what you want to see is each design plotted according to its merits, what I’ll call the "objective space". [after Mueller]
```
Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:41]: link to Mueller?

Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:41]: previous two paragraphs could be shortened I think

File: _posts/2015-10-08-design-is-multi-objective.md [hunk 4]
```diff
@@ -29,27 +29,27 @@ but inaccuracies in the model can result in absurdly unrealistic optima that are
...
 If you actually do prefer the "worse" design, this is a modeling error;
 either you have recognized a missing objective,
 or the second design is unrealistic and you need to further constrain your model,
 or it’s actually an unmodelable subjective, and should be broken down into multiple objectives.
 
-The subset of the objective space with only designs that are not strictly worse than another design is called the “Pareto frontier”,
+The subset of the objective space with only designs that are not strictly worse than another design is called the "Pareto frontier",
 and as the frontier of solutions possible under your model, it’s where the consideration of tradeoffs begins.
 By removing all these subpar designs, the Pareto frontier reduces the number of dimensions a designer has to consider from all variables to just those that are objectives.
 Objectives, by their nature, are not just fewer in number but also directly meaningful,
```
Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:55]: this para is good

Comment (whoburg) [inline _posts/2015-10-08-design-is-multi-objective.md:55]: the post ends pretty abruptly.

**Comment by whoburg** [comment]
this PR is ready for review.

**Comment by bqpd** [comment]
Oh, this was originally just a placeholder post to check formatting. Happy to edit it for actual use, though!

**Comment by whoburg** [comment]
Oh good - understood. I think this could turn into a good post but we should edit/improve it before it goes live.

Therefore, let's remove it for now. Is there a good way to hide it (but retain the edits)? Perhaps we move the file from `_posts/` to a new directory, `_drafts` ?

**Comment by bqpd** [comment]
Sounds like a very sensical way to do it. We should also remove the other placeholder posts, replacing them with your State of the Lab Address...and what's to be done with the homepage?

**Comment by whoburg** [comment]
ok,
- just pushed a commit implementing the move to `_drafts`
- other placeholder posts have been removed in #3 
- homepage still needs some basic info on the group's research interests, but that's outside the scope of this PR

I think this PR is ready to be merged -- it basically makes a few tweaks to the multiobjective post and hides it in `_drafts` to be edited/improved some more.
